resources/assets/coffee/react/profile-page/extra.coffee

```diff
@@ -103,4 +103,4 @@ class ProfilePage.Extra extends React.Component
       el ProfilePage.RecentActivities, recentActivities: @props.recentActivities
       el ProfilePage.Kudosu, user: @props.user, recentlyReceivedKudosu: @props.recentlyReceivedKudosu
       el ProfilePage.TopRanks, user: @props.user, scoresBest: @props.scoresBest, scoresFirst: @props.scoresFirst
-      el ProfilePage.Beatmaps, favouriteBeatmaps: @props.user.favouriteBeatmaps, rankedAndApprovedBeatmaps: @props.user.rankedAndApprovedBeatmaps
+      el ProfilePage.Beatmaps, favouriteBeatmapSets: @props.user.favouriteBeatmapSets, rankedAndApprovedBeatmapSets: @props.user.rankedAndApprovedBeatmapSets
```

Comment:
I just remembered why I don't use `@props.user.scoresBest` etc (and that weird variables passing in `profile-page.coffee`: when updating cover, only minimum set of user data is returned (`$user->defaultJson()`).